### PR TITLE
Remove test workaround for the limited access modals

### PR DESCRIPTION
**Pull request by KKoukiou**
…ase 231"

This reverts commit 75a9eb08dd3770782529a08f801609e2df907754.

It was meant to be a temporary workaround anyway.

### CODE REVIEW

**Review by marusak — DISMISSED** [review]
Thanks! I am of course fine with this as long as all tests are green :)

**Comment by marusak** [comment]
ubuntu-stable in travis is the one place where this one has not yet been fixed. I wonder when it will get there...

**Comment by KKoukiou** [comment]
@marusak Do you know what's wrong with this one? All tests passed by travis failed by showing that tests failed. I even retried it but did not change anything.

**Comment by marusak** [comment]
The image for ubuntu-stable that we use in travis does not yet have adequate cockpit version.

**Review by martinpitt — APPROVED** [review]
Looks good, and I'm fine with not blocking on ubuntu-stable for now. Now that the new development series is open, I can do backports again, I'll do that now. And then I'll send a PR to adjust travis to pull from backports.

**Comment by martinpitt** [comment]
I [backported 231](https://launchpad.net/ubuntu/+source/cockpit) to 20.10 and 20.04. Will take a few hours to appear on the mirrors, then we can pull it in. After that it actually ought to be enough to bump the cockpit-{bridge,system} dependencies in packaging/debian/control, everything else should then Just Work™. I'll test this tomorrow or Monday.

**Review by martinpitt — CHANGES_REQUESTED** [review]
Needs a packaging fix to bump cockpit dependency

**Comment by martinpitt** [comment]
It's already on the mirrors (I forgot how fast this is in Ubuntu). It still did need a little vm.install tweak, as apt doesn't use backports automatically to satisfy versioned dependencies.

**Comment by martinpitt** [comment]
Meh, tests on e2e fail due to the docker pull rate limit. But at least Travis is happy :-)

So it seems we need to fix/push PR #602 ASAP

File: src/manifest.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "version": "0.1",
     "requires": {
-        "cockpit": "137"
+        "cockpit": "231"
```
Comment (marusak) [inline src/manifest.json:4]: strictly speaking, it is not really required. The page would work just fine with 230 for example. Even though I don't really think 137 is adequate version, but this one seem rather strict for no obvious reason.

File: packaging/debian/control [hunk 2]
```diff
@@ -10,8 +10,8 @@ Package: cockpit-podman
 Architecture: all
 Multi-Arch: foreign
 Depends: ${misc:Depends},
-         cockpit-bridge (>= 138),
-         cockpit-system (>= 138),
+         cockpit-bridge (>= 231),
```
Comment (marusak) [inline packaging/debian/control:13]: The same reason as for manifest.

File: src/manifest.json (hunk 1 above)
Comment (martinpitt) [reply src/manifest.json]: Hmm, good point, this really was for the tests only. That was a thinko then. Reverted.

**Comment by martinpitt** [comment]
Argh, commit message was out of sync, updated.

**Review by marusak — APPROVED** [review]

**Pull request merged**